_submissions/round-12/3/2015-3-28-andrew-macdonald.md

```diff
@@ -0,0 +1,42 @@
+---
+data: '2015-03-28'
+round: Round 12
+title: Demotivation
+author: Andrew MacDonald
+permalink: 2015/3/andrew-macdonald-motivation
```

Comment:
Needs slashes before and after and a two-digit month, as in `/2015/03/andrew-macdonald-motivation/`.  Yeah, it's silly, but that's what Jekyll needs...